lib/laforge.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails/all'
+require 'laforge/data_entry'
+require 'laforge/data_source'
 require 'laforge/activerecord'
 
 module LaForge
 end
+
+ActiveSupport.on_load :active_record do
```

Comment:
I wonder if we should use `to_prepare` so this gets run on boot and whenever code is reloaded in development.
https://guides.rubyonrails.org/configuring.html#initialization-events

Reply:
Why would i want this to run whenever code is reloaded in development? Once the gem is installed I want to have access to LaForge ActiveRecord (ie. any model can be `laforged`)